stable/concourse/templates/worker-statefulset.yaml

```diff
@@ -28,7 +28,9 @@ spec:
                       "values": ["{{ template "worker.fullname" . }}"]
                     }]
                   },
-                  "topologyKey": "kubernetes.io/hostname"
+                  "podAffinityTerm": {
+                    "topologyKey": "kubernetes.io/hostname"
```

Comment:
How about removing the deprecated annotation as was done e. g. here:
https://github.com/kubernetes/charts/pull/1326/files